server.py

```diff
@@ -460,6 +486,28 @@ def delete(self, region, id, region_to_change):
 
         return convert_tournament_to_response(dao.get_tournament_by_id(tournament.id), dao)
 
+class FinalizeTournamentResource(restful.Resource):
+    """ Converts a pending tournament to a tournament.
+        Works only if the PendingTournament's alias_to_id_map is completely filled out.
+        Route restricted to admins for this region. """
+    def post(self, region, id):
+        dao = Dao(region, mongo_client=mongo_client)
+        user = get_user_from_access_token(request.headers, dao)
+
+        pending_tournament = dao.get_pending_tournament_by_id(ObjectId(id))
+        if not pending_tournament:
+            return 'No pending tournament found with that id.', 400
+        elif not is_user_admin_for_regions(user, pending_tournament.regions):
+            return 'Permission denied', 403
+        
+        try:
+            tournament = Tournament.from_pending_tournament(pending_tournament)
+            tournament_id = dao.insert_tournament(tournament)
+            dao.delete_pending_tournament(pending_tournament)
+            return {"success": True, "tournament_id": str(tournament_id)}
```

Comment:
Now that I think about it, we should keep the same ID when we convert the tournament from pending to real. On the frontend, everything appears under the Tournaments list, so it would be weird to all of a sudden swap the ID out after you finalize it. It also breaks links to that tournament.